idutils/__init__.py

```diff
@@ -16,1005 +16,26 @@
 
 """Small library for persistent identifiers used in scholarly communication."""
 
-from __future__ import absolute_import, print_function
+import importlib
```

Comment:
In general, I am ok with this change, but it might break imports of existing modules that use this. Or will it work?